tests/core/test_table.py

```diff
@@ -32,16 +33,14 @@ def df_from_list(tables: list[Table]):
     )
 
 
-@all_platforms
 class TestCoreTable:
-    def test_create_table(self, test_mp, request):
-        test_mp: Platform = request.getfixturevalue(test_mp)  # type: ignore
-        run = test_mp.runs.create("Model", "Scenario")
+    def test_create_table(self, platform: ixmp4.Platform):
+        run = platform.runs.create("Model", "Scenario")
 
         # Test normal creation
         indexset, indexset_2 = tuple(
-            IndexSet(_backend=test_mp.backend, _model=model)
-            for model in create_indexsets_for_run(platform=test_mp, run_id=run.id)
+            IndexSet(_backend=platform.backend, _model=model)  # type: ignore
+            for model in create_indexsets_for_run(platform=platform, run_id=run.id)
```

Comment:
Mostly to understand for myself: have you now reverted the introduction of this util function? If so, this is just to ensure it's not forgotten in these tests, either.

Reply:
No not this one as it was introduced after i started this branch (before my vacation a few weeks ago)